modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/TagsRatingTestCase.java

```diff
@@ -0,0 +1,131 @@
+/*
+ *Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
old license header

Reply:
check and fix all files

Reply:
Fixed with https://github.com/wso2/product-apim/pull/11163/commits/909e61ea2edc8a1e3f88aead2a5213324d94bd84